MAUI/DataGrid/Getting-started.md

```diff
@@ -8,48 +8,33 @@ documentation: ug
 keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .net maui datagrid, .net maui grid, .net grid maui
 ---
 
-# Getting Started with .NET MAUI DataGrid (SfDataGrid)
+# Getting Started with .NET MAUI DataGrid
 
-This section provides a quick overview for working with the SfDataGrid for .NET MAUI. Walk through the entire process of creating a real world of this control.
+This section provides a quick overview for working with the [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) for .NET MAUI.  Follow the steps below to add a basic DataGrid to your project.
 
 To get start quickly with .NET MAUI DataGrid, you can check on this video:
 
 <style>#MAUIDataGridVideoTutorial{width : 90% !important; height: 400px !important }</style> <iframe id='MAUIDataGridVideoTutorial' src='https://www.youtube.com/embed/xW0a7JlHbz4'></iframe>
 
-## Creating an application using the .NET MAUI DataGrid
- 1. Create a new .NET MAUI application in Visual Studio.
- 2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfDataGrid to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.DataGrid and then install it.
- 3. Import the control namespace `Syncfusion.Maui.DataGrid` in XAML or C# code.
- 4. Initialize the [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) control.
+## Prerequisites
 
- {% tabs %}
-{% highlight xaml tabtitle="MainPage.xaml" %}
+Before proceeding, ensure the following are in place:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
 
-<ContentPage   
-    . . .
-    xmlns:syncfusion="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid">
+## Step 1: Create a New MAUI Project
```

Comment:
Step 1: Create a New MAUI Project

Visual Studio

Go to File > New > Project and choose the .NET MAUI App template.
Name the project and choose a location, then click Next.
Select the .NET framework version and click Create.
Visual Studio Code

Open the command palette by pressing `Ctrl+Shift+P`` and type .NET:New Project and enter.
Choose the .NET MAUI App template.
Select the project location, type the project name and press enter.
Then choose Create project

Reply:
Modified